src/proto/thread_telemetry.proto

```diff
@@ -337,6 +343,9 @@ message TelemetryData {
 
     // The counters of response codes sent by the SRP server
     optional SrpServerResponseCounters response_counters = 6;
+
+    // The state of upstreamdns query
+    optional UpstreamDnsQueryState upstream_dns_query_state = 7;
```

Comment:
Why are we putting it in `SrpServerInfo`?

Reply:
Moved! Thanks!